README.md

```diff
@@ -5,23 +5,251 @@
-use of the [istio/installer](https://github.com/istio/installer) charts.
-1.   Provide validation of all parameters.
+Istio operator is not yet ready for users to try out (it will be shortly, stay tuned on discuss.istio.io). 
+However, the code is stable enough for developers to evaluate and experiment with. Contributions are very welcome - 
+see [open issues](), create a feature proposal, or come to the weekly Environments Working Group meeting if you have ideas. 
+
+### Background
+
+The current [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) have been reorganized into two groups:
+
+1.  A new [platform level installation API](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto), dealing with k8s settings like resources, auto scaling, pod disruption budgets etc. 
+1.  A configuration API, dealing with Istio control plane configuration settings. This API currently uses the [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) for backwards compatibility, but will be reorganized and slimmed down in the future.
+
+Some parameters will therefore temporarily be in both APIs - for example, setting k8s resources can be done through either API above. However, it's recommended to use the first API, since this has a more consistent structure and it is the one that will remain going forward.  
+Typical helm values parameter sets are currently provided as [profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for an Istio install, and can currently be customized by editing the values files or passing parameters when calling helm.  
+Similarly, the operator API uses the same profiles, which can be selected as a starting point for the installation.   
+For comparison, here is an installation of the SDS profile using helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+The same installation would be expressed as a CustomResource (CR) through the new API as:
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+The profile can be installed into the cluster by installing the operator into a pod into the cluster and applying the above CR, or applying it via the CLI mode. When using CLI mode, it's not necessary to specify the entire CR, only the spec portion i.e.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+In the absence of a profile, the default demo profile is installed. All profiles are available compiled in, or the profile can point to a local file path.
```

Comment:
```suggestion
If you don't specify a configuration profile, Istio is installed using the default `demo` configuration profile. 
```
The following sentence is very confusing, please rewrite: "All profiles are available compiled in, or the profile can point to a local file path."